src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -74,9 +74,9 @@
   private String gameName;
   private Version gameVersion;
   private int diceSides;
-  private transient List<TerritoryListener> territoryListeners = new CopyOnWriteArrayList<>();
-  private transient List<GameDataChangeListener> dataChangeListeners = new CopyOnWriteArrayList<>();
-  private transient List<GameMapListener> gameMapListeners = new CopyOnWriteArrayList<>();
+  private transient List<TerritoryListener> territoryListeners = new ArrayList<>();
+  private transient List<GameDataChangeListener> dataChangeListeners = new ArrayList<>();
+  private transient List<GameMapListener> gameMapListeners = new ArrayList<>();
```

Comment:
Good point. I don't think I went over this one closely enough. Any non-UI classes have the potential to be accessed by multiple threads. We may need to revert this PR.